config/locales/errors/en.yml

```diff
@@ -115,7 +115,8 @@ en:
       attestation_error: Sorry, but your security key doesn’t appear to be a FIDO
         security key.  Please make sure your device is listed at
         https://fidoalliance.org/certification/fido-certified-products/ and if
-        you believe the error is ours, please contact us at hello@login.gov.
+        you believe the error is ours, please contact at
+        https://login.gov/contact/
```

Comment:
Could we consider using the `MarketingSite.contact_url` method here, so we have a centralized place to manage it?